modules/core/src/test/java/org/apache/ignite/internal/hlc/HybridClockTest.java

```diff
@@ -75,13 +75,13 @@ public void testTick() {
         assertTimestampEquals(100, new HybridTimestamp(100, 2),
                 () -> clock.update(new HybridTimestamp(60, 1000)));
 
-        assertTimestampEquals(200, new HybridTimestamp(200, 0),
+        assertTimestampEquals(200, new HybridTimestamp(100, 3),
```

Comment:
Why this changes are done?
Clock behavior should remain the same.

Reply:
I do not think that the change violates the clock protocol but increases performance.

Reply:
I don't understand. That is the reason to change test?

Reply:
The reason is we can get the ts bigger than that is provided in parameters by incrementing the old value regardless of the physical ts.